src/main/scala/com/raquo/airstream/core/EventStream.scala

```diff
@@ -348,6 +349,31 @@ object EventStream {
     new JsPromiseStream[A](promise, emitOnce)
   }
 
+  def fromFlowPublisher[A](publisher: Flow.Publisher[A], emitOnce: Boolean = false): EventStream[A] = {
```

Comment:
Bikeshed?

```suggestion
  def fromPublisher[A](publisher: Flow.Publisher[A], emitOnce: Boolean = false): EventStream[A] = {
```

Reply:
👍 Fine by me, I don't think I'll ever have a `Publisher` type in Airstream